agenta-web/src/components/Playground/ViewNavigation.tsx

```diff
@@ -165,31 +180,39 @@ const ViewNavigation: React.FC<Props> = ({
                         <p>
                             Error connecting to the variant {variant.variantName}.{" "}
                             {(axios.isAxiosError(error) && error.response?.status === 404 && (
-                                <span>Container is not running.</span>
+                                <span>
+                                    Container is not running. <b>See logs below:</b>
+                                </span>
                             )) || <span>{error.message}</span>}
                         </p>
-                        <p>To debug this issue, please follow the steps below:</p>
                         <ul>
-                            <li>
-                                Verify whether the API is up by checking if {apiAddress} is
-                                accessible.
-                            </li>
-                            <li>
-                                Check if the Docker container for the variant {variantDesignator} is
-                                running. The image should be called {imageName}.
-                            </li>
+                            {isDemo() && (
```

Comment:
1. I don't see the difference here between isDemo and is !isDemo
2. Let's use the antd typography code style and style the logs container a bit
maybe @bekossy can help there 
https://ant.design/components/typography 
<img width="637" alt="Screenshot 2024-05-07 at 18 53 03" src="https://github.com/Agenta-AI/agenta/assets/6608260/53cd9bb4-f341-418f-a75c-e8ef00fc10a0">

Reply:
> I don't see the difference here between isDemo and is !isDemo

Why not? See code [here](https://github.com/Agenta-AI/agenta/blob/d4f93258bc018d44954c26605577921c5431a4d3/agenta-web/src/lib/helpers/utils.ts#L106-L111). The first condition will display for users in cloud and ee, while the second only for oss.